pyam_analysis/core.py

```diff
@@ -765,21 +787,15 @@ def read_ix(ix, regions=None):
     regions: list
         list of regions to be loaded from the database snapshot
     """
-    if not has_ix:
-        error = 'this option depends on the ixmp package'
-        raise SystemError(error)
     if isinstance(ix, ixmp.TimeSeries):
-        df = ix.timeseries()
+        df = ix.timeseries(iamc=False, regions=regions, variables=variables,
```

Comment:
the docs say this could also take a `Scenario`, is that no longer true?

Reply:
`Scenario` is a subclass of `TimeSeries`, so this should return `True` in either case (I think?). I included both ixmp classes explicitly in the docs because I don't think that too many users will ever come in contact with the `TimeSeries` class.